README_NetCore.md

```diff
@@ -125,8 +126,32 @@ To set your API credentials for an API request, configure the following informat
     keyPassword         = your_merchant_id
     keyFileName         = your_merchant_id
     keysDirectory       = Resource
+    useMetaKey          = false
     ```
 
+* MetaKey Http
+
+  ```
+    authenticationType  = http_Signature
+    merchantID          = your_child_merchant_id
+    merchantKeyId       = your_metakey_serial_number
+    merchantsecretKey   = your_metakey_shared_secret
+    portfolioId         = your_portfolio_id
+    useMetaKey          = true
+  ```
+
+* MetaKey JWT
+
+  ```
+    authenticationType  = Jwt
+    merchantID          = your_child_merchant_id
+    keyAlias            = your_child_merchant_id
+    keyPassword         = your_portfolio_id
+    keyFileName         = your_portfolio_id
+    keysDirectory       = Resource
+    useMetaKey          = true
+  ```
+
```

Comment:
Content for OAuth is missing. Copy it from Samples for Java.

Reply:
Done